### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by ark2002**
- Success Toast
- Error Toast

Preview:
Toast _ Astro UI

### CODE REVIEW

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    flex-wrap: wrap;
+    gap: 1rem;
+    padding: 2rem;
+    align-items: center;
+}
+
+.success__toast,.error__toast{
+    align-items: center;
+    color: var(--online-color);
+    background-color: #00ff002f;
```
Comment (ravikumar0403) [inline components/toast/toast.css:11]: use variable for this color as well

**Review by ravikumar0403 — APPROVED** [review]
Looks good to merge

**Review by Ronak-Mutha — APPROVED** [review]
Code looks clean and good to merge. Just one suggestion, you can use variable for color 'ff00002f' as well as it's getting repeated multiple times.

**Review by srv-pranjal — APPROVED** [review]
- Good to see that you implemented a demo button to show functionality. 
 - The code looks clean. Good work.
 - One suggestion - Display the JS code also on the UI for the user to implement toast easily.

**Pull request merged**